cmd/plasmacli/config/connection.go

```diff
@@ -28,42 +25,27 @@ func GetContractConn() (*eth.Plasma, error) {
 }
 
 func setupContractConn() (*eth.Plasma, error) {
-	// Parse plasma.toml before every call to the eth command
-	// Update ethereum client connection if params have changed
-	plasmaConfigFilePath := filepath.Join(viper.GetString(store.DirFlag), "plasma.toml")
-
-	if _, err := os.Stat(plasmaConfigFilePath); os.IsNotExist(err) {
-		plasmaConfig := DefaultPlasmaConfig()
-		WritePlasmaConfigFile(plasmaConfigFilePath, plasmaConfig)
-	}
-
-	viper.SetConfigName("plasma")
-	if err := viper.MergeInConfig(); err != nil {
+	conf, err := ParseConfigFromViper()
+	if err != nil {
 		return nil, err
 	}
 
-	conf, err := ParsePlasmaConfigFromViper()
-	if err != nil {
-		return nil, err
+	dir := viper.GetString(flags.Home)
+	if dir[len(dir)-1] != '/' {
+		dir = dir + "/"
 	}
 
-	// Check to see if the eth connection params have changed
-	dir := viper.GetString(store.DirFlag)
 	if conf.EthNodeURL == "" {
-		return nil, fmt.Errorf("please specify a node url for eth connection in %s/plasma.toml", dir)
+		return nil, fmt.Errorf("please specify a node url for eth connection in %sconfig.toml", dir)
 	} else if conf.EthPlasmaContractAddr == "" || !ethcmn.IsHexAddress(conf.EthPlasmaContractAddr) {
-		return nil, fmt.Errorf("please specic a valid contract address in %s/plasma.toml", dir)
+		return nil, fmt.Errorf("please specify a valid contract address in %sconfig.toml", dir)
 	}
 
 	ethClient, err := eth.InitEthConn(conf.EthNodeURL)
 	if err != nil {
 		return nil, err
 	}
-	blockFinality, err := strconv.ParseUint(conf.EthBlockFinality, 10, 64)
-	if err != nil {
-		return nil, err
-	}
-	plasma, err := eth.InitPlasma(ethcmn.HexToAddress(conf.EthPlasmaContractAddr), ethClient, blockFinality)
+	plasma, err := eth.InitPlasma(ethcmn.HexToAddress(conf.EthPlasmaContractAddr), ethClient, 0)
```

Comment:
why is blockFinality 0 here? I think we should support rejecting deposits that haven't passed the finality bound

Reply:
like maybe we should update the cli commands to support this

Reply:
Is there a command in the cli that would make sense to include this?

I think it should be removed the config parameter to avoid confusion until it is reintroduced

Reply:
`include-deposit` and the query deposit in eth could

Reply:
but yea we can open up an issue, link this pr so its easy to copy paste the config back in

Reply:
created an issue for it